File: scripts/problems/fibonacci/boilerplate-full.js
const lines = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
let _idx = 0;
const nextLine = () => lines[_idx++];

// ── USER CODE START ──
{{USER_CODE}}
// ── USER CODE END ──

const n = parseInt(nextLine());

const result = fib(n);
console.log(result);

File: scripts/problems/fibonacci/boilerplate-full.rs
use std::io::{self, BufRead};

fn next_line() -> String {
    let stdin = io::stdin();
    let mut lines = stdin.lock().lines();
    lines.next().unwrap().unwrap()
}

// ── USER CODE START ──
{{USER_CODE}}
// ── USER CODE END ──

fn main() {
    let n: i64 = next_line().trim().parse().unwrap();

    let result = fib(n);
    println!("{}", result);
}